data/containers/helm-test/templates/job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-job"
+spec:
+  template:
+    spec:
+      containers:
+      - image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+        name: helm-test-{{ .Values.job_id }}
+        command: [ "cat", "/etc/os-release" ]
+        resources:
```

Comment:
Should be nice to have the limits on variables with a default value

Reply:
Do you think that this is needed? The variables are defined just in another file.